content/_data/changelogs/weekly.yml

```diff
@@ -14565,6 +14565,125 @@
   # pull: 6313 (PR title: Clarify that the changelog entry should be in the imperative mood)
   # pull: 6314 (PR title: Bump script-security from 1138.v8e727069a_025 to 1140.vf967fb_efa_55a_)
 
+  - version: '2.338'
+    date: 2022-03-08
+    changes:
+      - type: rfe
+        category: developer
+        pull: 6317
+        authors:
+          - glasswalk3r
+          - 'TODO: alceu.freitas (alceu.freitas@semantix.com.br)'
+        pr_title: Refactor `translation-tool.pl`
+        message: |-
+          Developer: Add a <code>--debug</code> option to the translation tool to help in troubleshooting. Improve the readability of the translation tool command-line interface (CLI).
+      - type: TODO
+        category: developer
+        pull: 6322
+        authors:
+          - dependabot[bot]
+        pr_title: Bump Guava from 31.0.1 to 31.1
+        message: |-
+          Upgrade the [Guava](https://guava.dev/) library from [31.0.1](https://github.com/google/guava/releases/tag/v31.0.1) (released September 27, 2021) to [31.1](https://github.com/google/guava/releases/tag/v31.1) (released February 28, 2022).
+      - type: rfe
+        category: rfe
+        pull: 6229
+        authors:
+          - NotMyFault
+        pr_title: Use modern icons in workspace view of a job
+        message: |-
+          Use modern icons at the workspace view.
+      - type: rfe
+        category: rfe
+        pull: 6319
+        authors:
+          - glasswalk3r
+        pr_title: Add Brazilian Portuguese translation property files
+        message: |-
+          Add Brazilian Portuguese translation property files.
+      - type: bug
+        category: regression
+        pull: 6342
+        authors:
+          - basil
+        pr_title: Stop overemphasizing console logs
+        message: |-
+          Stop overemphasizing console logs (regression in 2.335).
+      - type: bug
+        category: bug
+        pull: 6340
+        issue: 67965
+        authors:
+          - janfaracik
+        pr_title: "[JENKINS-67965] Fix checkbox focus state"
+        message: |-
+          Checkbox focus state restored
```

Comment:
Please cross-check the tone of the sentence.
And it's missing a full stop at the end.